settings/production.py

```diff
@@ -15,11 +15,26 @@
 SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 
 # Allow production host headers
-ALLOWED_HOSTS = ['example.pennlabs.org']
+ALLOWED_HOSTS = [BACKEND_DOMAIN]
 
 SENTRY_URL = os.environ.get('SENTRY_URL', '')
 
 sentry_sdk.init(
     dsn=SENTRY_URL,
     integrations=[DjangoIntegration()]
 )
+
+###############################################################
+# SETTINGS TO ALLOW FRONTEND TO MAKE AJAX REQUESTS TO BACKEND #
+###############################################################
+# DO NOT USE IF DJANGO APP IS STANDALONE
+# Django CORS Settings
+CORS_ORIGIN_REGEX_WHITELIST = [
```

Comment:
We should probably switch to non-regex and just explicitly define the two valid entries.

Reply:
Why? I think the regex is clear enough, and it's more flexible if we want a prefix that's not `api`.

Reply:
The CORS whitelist is for whatever the frontend domain is, so the backend domain doesn't matter. I don't see the value in matching `w.{FRONTEND_DOMAIN}`, `www.{FRONTEND_DOMAIN}`, and `wwwwwwwwww.{FRONTEND_DOMAIN}` when we really only want `www`.

Reply:
Oh fair